docs/standards/rest.md

```diff
@@ -201,6 +201,44 @@ The `version` URL query parameter, `?version=version_string` is reserved for sel
 
 `starting_after`, `ending_before`, and `limit` are reserved for cursor pagination on resource collections, as defined in [JSON API Pagination Parameters](../../principles/jsonapi/#pagination-parameters).
 
+### <a id="expansion"></a>Expansion
+
+A resource's relationships may inline the related resource's attributes to conserve API requests and provide a richer response. This inline enrichment of related data attributes is called _expansion_.
+
+#### `expand` query parameter
+
+The `expand` URL query parameter is reserved for expressing expansion of relationships. Resource endpoints are not required to support this parameter, but when they do, they must declare the `expand` parameter schema as an array of enum values. The enum values must match the relationship names provided in the resource data `relationships` map.
+
+The parameter should be declared in OpenAPI with the same representation as other array parameters, `{"style": "form", "explode": false}`.
```

Comment:
Can we get a rule to enforce this? If it's not in code, it's not really a standard!

Reply:
Yes, in a followup.